src/representation_theory/basic.lean

```diff
@@ -374,6 +374,12 @@ def dual : representation k G (module.dual k V) :=
 @[simp]
 lemma dual_apply (g : G) : (dual ρV) g = module.dual.transpose (ρV g⁻¹) := rfl
 
+/--
+Given `k`-modules `V, W`, there is an isomorphism `φ : V^* ⊗ W → Hom_k(V, W)`.
+Given representations of `G` on `V` and `W`,there are representations of `G` on `V^* ⊗ W` and on
+`Hom_k(V, W)`.
+This lemma says that `φ` is `G`-linear.
+-/
```

Comment:
Can you expand this docstring to explain where `φ` appears in the lemma?

Reply:
thanks, added a new commit doing this